### DISCUSSION TITLE
Missing Gist's (as a tab) on the profile page

### DISCUSSION THREAD

**Discussion by michitheonlyone** [community/community#30750] [Other Feature Feedback, Questions, & Ideas]
Everything related to a user has a tab in the profile overview except for "gist"?
@github-staff coul you add gist as a tab beside the repos would be nice ;-)

**Comment by thomas-ayissi** [comment]
Hello,
While waiting for the option to be added. have you tried to give stars to the Gist you like or use often?
Giving stars is an easy and convenient way to find Gists.
Have a nice weekend
